lib/api_repo.js

```diff
@@ -231,14 +231,22 @@ ApiRepo.prototype.buildPackages =
                 cleanName.replace(new RegExp('-', 'g'), '/');
             opts.packageInfo.api.name = that.pkgPrefix + cleanName;
             opts.packageInfo.api.version = version;
-            opts.packageInfo.api.buildCommonProtos =
-                that.opts.buildCommonProtos;
```

Comment:
I think `buildCommonProtos` option should be kept here. That's to create a new common-protos package and doesn't conflict with your change by itself.